tools/localization/src/update.ts

```diff
@@ -0,0 +1,200 @@
+/**
+ * @author
+ * AnkiDroid Open Source Team
+ * 
+ * @license
+ * Copyright (c) AnkiDroid. All rights reserved.
+ * Licensed under the GPL-3.0 license. See LICENSE file in the project root for details.
+ * 
+ * @description
+ * updateI18nFiles() to update files in AnkiDroid/src/main/res/values/ with downloaded files from crowdin. 
+ * The downloaded file needs to extract first with 'yarn start extract'.
+ * It's expected to be called through 'yarn start update'.
+ */
+
+import fs from 'fs';
+import path from "path";
+import readline from 'readline';
+import { LANGUAGES, LOCALIZED_REGIONS, TEMP_DIR, TITLE_STR, TITLE_FILE, I18N_FILES, XML_LICENSE_HEADER, RES_VALUES_LANG_DIR, OLD_VER_MARKET_DESC_FILE, MARKET_DESC_LANG } from "./constants";
+
+let anyError = false;
+
+/**
+ * Replace invalid chars in xml files in res/value dir
+ * e.g. %1s$ is invalid, %1$s is valid
+ * 
+ * @param fileName name of the file in res/value dir
+ * @returns correct/replaced strings in the files
+ */
+async function replacechars(fileName: string) {
```

Comment:
Arthur noticed this the casing, I also propose strongly typing the return value, I think it is boolean?

```suggestion
async function replaceChars(fileName: string): boolean {
```